src/vs/workbench/contrib/search/browser/searchView.ts

```diff
@@ -85,6 +87,43 @@ export enum SearchViewPosition {
 	Panel
 }
 
+export type SearchViewContextMid = 13;
+export const SearchViewContextMid = 13;
```

Comment:
Still unsure what the 13 actually does?  Is it a order / position?

Reply:
It's an ID used to tag data transfer object